staging/src/k8s.io/apiserver/pkg/apis/flowcontrol/bootstrap/default.go

```diff
@@ -89,6 +89,10 @@ var (
 		flowcontrol.PriorityLevelConfigurationNameExempt,
 		flowcontrol.PriorityLevelConfigurationSpec{
 			Type: flowcontrol.PriorityLevelEnablementExempt,
+			Exempt: &flowcontrol.ExemptPriorityLevelConfiguration{
+				NominalConcurrencyShares: 0,
+				LendablePercent:          pointer.Int32(50),
```

Comment:
Given the default is 0 shares, then lendablePercent doesn't matter (any % from 0 is 0).
So I would just set it to 0 (or even leave empty) for simplicity.

Reply:
that sounds good to me, do you want me to open a PR to fix the KEP then? https://github.com/kubernetes/enhancements/pull/3906/files#diff-0d81f61bda1a026d678a311aa592601d41014434a5a3c91e17a3a3dfe1d4527aR830

Reply:
Probably yes.

Reply:
I may have missed this discussion, and would like some clarification here - we currently have the following:
a) we don't allow creation of a new `exempt` type prioritylevelconfiguration object
https://github.com/kubernetes/kubernetes/blob/544005c32d8eed925e35fa550eb5eca0b01b140d/pkg/apis/flowcontrol/validation/validation.go#L364-L366

b) we don't allow the admin to change the spec of the singleton mandatory `exempt` prioritylevelconfiguration object
https://github.com/kubernetes/kubernetes/blob/544005c32d8eed925e35fa550eb5eca0b01b140d/pkg/apis/flowcontrol/validation/validation.go#L348-L356

given `a` and `b`, are these new API meant to be tweaked by the apf maintainers only from release to release? If that's the case, we (the maintainers) can accomplish the same goal by making internal changes (without the proposed API) so the dispatching of `exempt` is no longer unlimited and has a relation with the dispatching for non-exempt priority levels.

Or are we going to relax `b` so the admin can tweak the singleton `exempt` pl?